src/fu-tool.c

```diff
@@ -2781,6 +2784,48 @@ fu_util_switch_branch (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete (priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_lock (FuUtilPrivate *priv, GError **error)
+{
+#ifdef HAVE_WRLCK
+	struct flock lockp = {
+		.l_type = F_WRLCK,
+		.l_whence = SEEK_SET,
+	};
+	g_autofree gchar *localstatedir = NULL;
+	g_autofree gchar *lockfn = NULL;
+
+	/* open file */
+	localstatedir = fu_common_get_path (FU_PATH_KIND_LOCALSTATEDIR);
+	lockfn = g_build_filename (localstatedir, "run", "fwuptool", NULL);
```

Comment:
`s/fwuptool/fwupdtool/`

Reply:
Thanks, fixed. I think the tool should probably have been called fwuptool and fwupmgr but we can't go back in time now :)